dateparser/date.py

```diff
@@ -14,7 +14,7 @@
 
 
 def sanitize_spaces(html_string):
-    html_string = re.sub(u'\xa0', ' ', html_string)
+    html_string = re.sub(u'\xc2\xa0', ' ', html_string)
```

Comment:
Agreed, `u''` prefix won't be needed. Yep, using `re.UNICODE` makes it work just fine.